classes/04class/exercises/c04-iac01/Faria-Marcio/s3.tf

```diff
@@ -0,0 +1,14 @@
+resource "random_string" "random" {
+  length  = 8
+  upper   = false
+  special = false
+}
```

Comment:
you're not using this random being created here. can you use it or remove it?

Reply:
Thanks for that Kiko. I removed it.